app/views/profile/_email_notifications.html.erb

```diff
@@ -9,13 +9,13 @@
     <%= f.form_group :email_notifications do %>
       <%= f.check_box :email_for_announcement,
                       label: t('profile.email_for_announcement'),
-                      custom: true %>
```

Comment:
The `custom` attribute comes from [`bootstrap_form`](https://github.com/bootstrap-ruby/bootstrap_form), I guess? Maybe an old version of it since in the current one I cannot find it either... Or is this related to something else? Searching for `custom` in the whole codebase doesn't reveal anything particularly interesting. It's also used in `app/views/profile/_subscription_type.html.erb` as well as `app/views/questions/_basics.html.erb`.

 I haven't found out why we use it and it didn't seem necessary, so I removed it. Please double-check this. The attribute was introduced around 4 years ago.

Reply:
Yes, this comes from bootstrap_form, it is no longer necessary. See also [the old REAME for the gem](https://github.com/bootstrap-ruby/bootstrap_form/blob/main/OLD-README.md).